sched/wdog/wd_initialize.c

```diff
@@ -50,29 +50,3 @@ clock_t g_wdtickbase;
 /****************************************************************************
  * Public Functions
  ****************************************************************************/
-
-/****************************************************************************
- * Name: wd_initialize
- *
- * Description:
- * This function initializes the watchdog data structures
- *
- * Input Parameters:
- *   None
- *
- * Returned Value:
- *   None
- *
- * Assumptions:
- *   This function must be called early in the initialization sequence
- *   before the timer interrupt is attached and before any watchdog
- *   services are used.
- *
- ****************************************************************************/
-
-void wd_initialize(void)
-{
-  /* Initialize watchdog lists */
-
-  sq_init(&g_wdactivelist);
```

Comment:
So we do nor need this call anymore? Why?

Reply:
Yes, please look sq_init definition:
https://github.com/apache/incubator-nuttx/blob/master/include/queue.h#L34-L40